internal/plugins/manifests/init.go

```diff
@@ -83,6 +83,9 @@ ifneq ($(origin DEFAULT_CHANNEL), undefined)
 BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+# this allows running the command with the container image instead
+# the command for that will be 'docker run --rm --workdir /root/tmp --interactive --volume $(pwd):/root/tmp quay.io/operator-framework/operator-sdk:v1.2.0'
```

Comment:
First, we cannot add `quay.io/operator-framework/operator-sdk:v1.2.0` since it will be outdated with a new release.
Then, why we would add this comment? Also, why `OPERATOR_SDK_CLI_COMMAND ?= operator-sdk` instead  of only `operator-sdk` 

c/c @estroz

Reply:
you are right, and I will remove comment.
I just wanted to make it easier to use for someone who wants to override it.

also as I mentioned before, this image is built as root to `operator-sdk generate` changes permissions of the folder

Reply:
sorry, it still not clear for me. The change here shows to be : 

Replace
```
operator-sdk generate kustomize manifests -q
```

With

```
$(OPERATOR_SDK_CLI_COMMAND) generate kustomize manifests -q
```

Where `$(OPERATOR_SDK_CLI_COMMAND)` is == `operator-sdk`. 

In this way: 

1. What did you do that is not achievable with the current code?
2. What did you expect to see? What did you see instead? Under which circumstances?
3. What problem it solves? What is not possible to do without using the var that will be possible to do with the var in the makefile?

Reply:
I want to run this code on a machine that doesn't have `operator-sdk` and needs to use a container to keep itself clean and without changes.
1. I cannot override the operator-sdk binary with something else, it's hardcoded
2. I expect it to run the same most times, but that I will be able to run `make bundle OPERATOR_SDK_CLI_COMMAND=...` in special use cases
3. I cannot do that without the envvar as I will need to create a new target only for this reason, which is something I don't want to do